client/lib/cart-values/types.ts

```diff
@@ -1,16 +0,0 @@
-/**
- * External dependencies
- */
-import {
-	ResponseCart,
-	ResponseCartProduct,
-	ResponseCartProductExtra,
-} from '@automattic/shopping-cart';
-
-// These types are deprecated. Please use types from @automattic/shopping-cart directly.
-
-export type CartItemValue = ResponseCartProduct;
-
-export type CartItemExtra = ResponseCartProductExtra;
-
-export type CartValue = ResponseCart;
```

Comment:
There are a number of "cart as `CartValue` object" comments in `/lib/analytics/`. Should they be updated to `ResponseCart`?

Reply:
👍  Thanks!